### PR TITLE
allow for list of timerpercentiles

### PR DESCRIPTION
**Pull request by nathanielc**

### CODE REVIEW

**Comment by txaj** [comment]
interesting way not to break the interface, however 1/ do we want to do that ? (be that loose with types) 2/ there's no docs

**Comment by ghoneycutt** [comment]
Looks good to me, but needs docs, spec tests and validation of `$timerpercentile`

File: templates/plugin/statsd.conf.erb [hunk 1]
```diff
@@ -17,7 +17,11 @@
 <% unless @deletesets.nil? -%>
   Deletesets <%= @deletesets %>
 <% end -%>
-<% if @timerpercentile -%>
+<% if @timerpercentile.kind_of?(Array) -%>
+<% @timerpercentile.each do |p| -%>
+  TimerPercentile <%= p %>
+<% end -%>
+<% elsif @timerpercentile -%>
```
Comment (igalic) [inline templates/plugin/statsd.conf.erb]: why not just,
```ruby
<% [@timerpercentile].flatten.compact.each do |p| -%>
  TimerPercentile <%= p %>
<% end -%>
```

**Comment by blkperl** [comment]
Closing due to lack of activity. Please reopen when the feedback has been addressed

**Pull request closed without merging**